rm-datasource/src/main/java/io/seata/rm/datasource/sql/serial/SerialArray.java

```diff
@@ -0,0 +1,190 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.seata.rm.datasource.sql.serial;
+
+import javax.sql.rowset.serial.SerialBlob;
+import javax.sql.rowset.serial.SerialClob;
+import javax.sql.rowset.serial.SerialDatalink;
+import javax.sql.rowset.serial.SerialException;
+import javax.sql.rowset.serial.SerialJavaObject;
+import java.net.URL;
+import java.sql.Blob;
+import java.sql.Clob;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.Map;
+
+/**
+ * used for jdbc type is JDBCType.ARRAY serialize.
+ *
+ * @author jsbxyyx
+ */
+public class SerialArray implements java.sql.Array, java.io.Serializable {
```

Comment:
Why not use javax.sql.rowset.serial.SerialArray?

Reply:
use javax.sql.rowset.serial.SerialArray to serializer framework may be throws exception.
details see: getResultSet method, no-constructor method.